libmscore/note.cpp

```diff
@@ -3053,7 +3053,7 @@ Element* Note::prevSegmentElement()
 //   lastTiedNote
 //---------------------------------------------------------
 
-const Note* Note::lastTiedNote() const
+Note* Note::lastTiedNote()
       {
       std::vector<const Note*> notes;
       const Note* note = this;
```

Comment:
If you remove `const` from this declaration there should be also no need in doing a `const_cast` a few lines later (at line 3069)

Reply:
Ah, yes, the implementation of `const` flavor of this function should also be corrected:
`const Note* lastTiedNote() const { return const_cast<Note*>(this)->lastTiedNote(); }